sensu_configs/init.d/sensu-service-wrapper

```diff
@@ -12,6 +12,7 @@
 # Description:    Sensu monitoring framework service script
 ### END INIT INFO
 
+short_service=$1
```

Comment:
maybe `short_service` should be `sensu_service` and `sensu_service` should be `service_name`?